<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TemplateSelectionPage</class>
 <widget class="QWidget" name="TemplateSelectionPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTreeView" name="treeView"/>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::ExpandingFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="identifierLabel">
       <property name="text">
        <string>&amp;Identifier</string>
       </property>
       <property name="buddy">
        <cstring>identifierLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="KLineEdit" name="identifierLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="directoryLabel">
       <property name="text">
        <string>&amp;Directory</string>
       </property>
       <property name="buddy">
        <cstring>directoryUrlRequester</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="KUrlRequester" name="directoryUrlRequester"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KFontRequester</class>
   <extends>QWidget</extends>
   <header>kfontrequester.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
